src/Conditional/LodashFunctions/Collection.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+
+namespace Northwestern\SysDev\DynamicForms\Conditional\LodashFunctions;
+
+
+trait Collection
+{
+    public static function includes($collection, $value, $fromIndex = 0)
```

Comment:
```suggestion
    public static function includes(string|object|array $collection, $value, $fromIndex = 0)
```

I think we have union types in 8.0?